models/codec/speechtokenizer/quantization/vq.py

```diff
@@ -0,0 +1,119 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
```

Comment:
Modify the copyright information

Reply:
these files are now removed, they now reference the encodec package instead of being files in Amphion